lib/v2/xinhua-daily-telegraph/daily.js

```diff
@@ -0,0 +1,71 @@
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+const { parseDate, getElementChildrenInnerText, getHTMLWithCache } = require('./utils');
+
+module.exports = async (ctx) => {
+    const date = timezone(new Date(), 8);
+    const dateFormatted = date.getFullYear().toString() + ('0' + (date.getMonth() + 1).toString()).slice(-2) + date.getDate().toString();
```

Comment:
You can use `dayjs` instead of manipulate the date string manually
```js
const dayjs = require('dayjs');
const utc = require('dayjs/plugin/utc');
dayjs.extend(utc);
const dateFormatted = dayjs().utcOffset(8).format('YYYYMMDD'));
```
---
or you can simply visit `http://mrdx.cn/content/PaperIndex.htm` since it will redirect you to the right place